packages/client/src/workspace/apps.ts

```diff
@@ -1,20 +1,32 @@
 import { App } from "@openfin/workspace"
 
-import { BASE_URL } from "./consts"
-import { manifestUrls } from "./utils"
+import { BASE_URL, manifestUrls } from "./constants"
+import { getCurrentUser, USER_TRADER } from "./user"
 
-export async function getApps(): Promise<App[]> {
-  return [
-    reactiveTraderFx,
-    reactiveTraderCredit,
-    reactiveAnalytics,
-    reactiveWorkspace,
-    limitCheckerView,
-    reactiveTraderFxLiveRatesView,
-    reactiveTraderFxTradesView,
-    reactiveTraderFxAnalyticsView,
-    reactiveAnalyticsView,
-  ]
+export function getApps(): App[] {
+  return getCurrentUser() === USER_TRADER
+    ? [reactiveTraderFx, reactiveTraderCredit, reactiveAnalytics]
+    : [reactiveAnalytics, limitChecker]
+}
+
+export function getViews(): App[] {
+  return getCurrentUser() === USER_TRADER
+    ? [
+        reactiveTraderFxLiveRatesView,
+        reactiveTraderFxTradesView,
+        reactiveTraderFxAnalyticsView,
+        reactiveAnalyticsView,
+      ]
+    : [
+        reactiveTraderFxTradesView,
```

Comment:
is there a reason that live rates had been excluded from here?

Reply:
yes, see previous categorisation in the store - the Home search was not categorised previously, but should have been if Store was restricted (obviously) .. we will need to verify the current state of the Workspace config with stakeholders/demoers but that's what we'll use UAT deployment for ..